cli/tests/common/test_environment.rs

```diff
@@ -94,6 +96,31 @@ impl Default for TestEnvironment {
 }
 
 impl TestEnvironment {
+    /// Runs `jj args..` at the `current_dir`, returns the output.
+    #[must_use = "either snapshot the output or assert the exit status with .success()"]
+    pub fn jj_at<I>(&self, current_dir: &Path, args: I) -> CommandOutput
```

Comment:
It seems like this is going to become the main entry point, right? `jj_at()` is not very intuitive to me. I guess it's not `jj_cmd()` just because that's already taken. How about `run_jj()` (and `run_jj_with()` for the other one)?

Reply:
Yes, and we might add something like `.at(dir) -> TestRepo<'_>; test_repo.jj(args)` later. `.jj_cmd()` won't be removed since there will be a few callers who don't wanna run command, but just get the command builder object.

I'm okay with renaming to `run_jj_at()`, `run_jj_with()`, and `new_jj_cmd()`.

Reply:
Sounds good. But `run_jj_in()` seems more more intuitive than `run_jj_at()`. At least I think of it as running in a directory more than running at a path.

Reply:
Perhaps, I inherited it from `*at()` POSIX functions. I'm okay with either way.

Reply:
I prefer `run_jj_in()`. The `_at()` made me think it was about running at a certain operation id. I'd get used to it quickly, of course, but it seems less intuitive.

Reply:
Maybe `_in` has implication that the environment is isolated within the directory? That wouldn't matter in this context, so `_in` seems also good.

Reply:
The plan is to add helper type that holds `repo_root`, and we'll set the root once and do `.run_jj(args)`. That's why I added `_in` suffix to the function that takes `cwd`.